modules/ROOT/pages/to-start-server-debug-mode.adoc

```diff
@@ -1,76 +1,82 @@
-= Starting the Mule Server In Debug Mode
+= Debug your Application on a Remote Mule Instance From Studio
 ifndef::env-site,env-github[]
```

Comment:
INSTRUCTIONS FOR EDIT REVIEW

Hi Kim! I have an edit request for a Studio example: https://github.com/mulesoft/docs-studio/pull/77/
I only need the file modules/ROOT/pages/to-start-server-debug-mode.adoc edited.
I updated pretty much the entire article, so please edit the whole thing.